jquery.flot.js

```diff
@@ -2415,7 +2422,7 @@ Licensed under the MIT license.
                     }
 
                     if (f.x === true) {
-                        if (val < range.xmin) {
+                        if (minCondition(val, range.xmin)) {
```

Comment:
In my opinion this is not the right way to do it. It would be better to change the previous condition - 

```
if (points[j] === null) {
    continue;
}
```

into something like - considerTheValueForAutoscale (but better named)
that you could overwrite